src/matchers/toBeEmpty/predicate.js

```diff
@@ -0,0 +1,3 @@
+import _ from 'lodash';
+
+export default value => _.isEmpty(value);
```

Comment:
Lodash's `isEmpty` seems to deem more as empty than this matcher should `_.isEmpty(1) // true`

The matcher should only consider:
`Use .toBeEmpty when checking if a String '', Array [] or Object {} is empty.`

I think for now it might be worth leaving lodash out